packages/transactions/src.ts/index.ts

```diff
@@ -356,7 +356,7 @@ export function serialize(transaction: UnsignedTransaction, signature?: Signatur
     });
 }
 
-export function serializeHederaTransaction(transaction: TransactionRequest) : HederaTransaction {
+export function serializeHederaTransaction(transaction: TransactionRequest, pubKey?: HederaPubKey) : HederaTransaction {
```

Comment:
Why do we need to add `pubkey` as another argument in the `serialiseHederaTransaction`?